tests/system/I18n/TimeDifferenceTest.php

```diff
@@ -19,14 +19,25 @@
  */
 final class TimeDifferenceTest extends CIUnitTestCase
 {
+    private string $currentLocale;
+
     protected function setUp(): void
     {
         parent::setUp();
 
         helper('date');
+
+        $this->currentLocale = Locale::getDefault();
         Locale::setDefault('America/Chicago');
```

Comment:
Parameter locale Is a BCP 47 compliant language tag. Is a **BCP 47** compliant language tag.
I thought the input of the function should be `en-US`, not `America/Chicago`!
https://www.php.net/manual/en/locale.setdefault.php

Reply:
Exactly.

Reply:
Fixed.